Engine_Revit_UI/Query/BottomLevel.cs

```diff
@@ -38,8 +38,8 @@ static public Level BottomLevel(this oM.Geometry.ICurve curve, Document document
                 return null;
 
             double aMinElevation = MinElevation(curve);//BH.Engine.Geometry.Query.Bounds(curve as dynamic).Min.Z;
-            if (convertUnits)
-                aMinElevation = UnitUtils.ConvertToInternalUnits(aMinElevation, DisplayUnitType.DUT_METERS);
+            //if (convertUnits)
```

Comment:
If this code isn't needed then let's remove it for tidy code? Unless it might be needed later?

Reply:
I prefer to keep it. I use BottomeLevel method in many places... If this change affect other method and causing trouble then I need to know how it worked before.

Reply:
Understood, in that case I'm happy with it :smile: